deploy.sh

```diff
@@ -29,6 +29,13 @@ then
         -e KBC_DEVELOPERPORTAL_PASSWORD \
         quay.io/keboola/developer-portal-cli-v2:latest \
         update-app-repository ${KBC_DEVELOPERPORTAL_VENDOR} ${KBC_DEVELOPERPORTAL_APP} ${IMAGE_TAG} ecr ${REPOSITORY}
+
+    # publish GELF variant of the component
```

Comment:
Nevim jestli si to myslel takle s tim, ze si budu muset tu komponentu pushnout i jako GELF variantu?

Reply:
jop přesně takhle